docs/concepts/07_dip/02_provider.md

```diff
@@ -0,0 +1,81 @@
+
+The pallet is agnostic over the chain-specific definition of identity and delegates the definition of it to the provider chain's runtime.
+
+Because providers and consumers evolve at different speeds, identity commitments are versioned.
+This allows the provider chain to upgrade to a newer commitment scheme while allowing users to use the old version if the chains on which they want to use their identity don't yet support the new scheme.
+
+The identity subject can replace or remove identity commitments, for example, if something in the identity info changes.
+After removal, the identity becomes unusable cross-chain, although it continues to exist on the provider chain and is usable for local operations.
+
+#### Add the pallet to the runtime
+
+Add the pallet to runtime in your `Cargo.toml` file dependencies section:
+
+:::info
+
+The DIP pallet is available in the KILT node release 1.12.0 and later.
+
+:::
+
+```toml
+provider = {package = "pallet-dip-provider", git = "https://github.com/KILTprotocol/kilt-node.git", branch = "release-1.12.0"}
+```
+
+#### Configure the pallet
+
+Pass runtime configuration to the pallet's `Config` trait.
+
+```rust
+impl pallet_dip_provider::Config for Runtime {
+    // Config
+}
+```
+
+<!-- TODO: Link for full details -->
+
+The following configuration parameters are required. Read the xxx :
+
+-   `type CommitOriginCheck: EnsureOrigin<Self::RuntimeOrigin, Success = Self::CommitOrigin>`: The check ensures a given runtime origin is allowed to generate and remove identity commitments.
+-   `type CommitOrigin: SubmitterInfo<Submitter = Self::AccountId>`: The resulting origin if `CommitOriginCheck` returns with errors. The origin is not required to be an `AccountId`, but must include information about the `AccountId` of the tx submitter.
+-   `type Identifier: Parameter + MaxEncodedLen`: The type of an identifier used to retrieve identity information about a subject.
+-   `type IdentityCommitmentGenerator: IdentityCommitmentGenerator<Self>`: The type responsible for generating identity commitments, given the identity information associated with a given `Identifier`.
+-   `type IdentityProvider: IdentityProvider<Self>`: The type responsible for retrieving the information associated with a subject given their identifier. The information can potentially be retrieved from any source, using a combination of on-chain and off-chain solutions.
+-   `type IdentityProvider: IdentityProvider<Self>`: Customizable external logic to handle events in which a new identity commitment is generated or removed.
+-   `type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>`: The aggregate `Event` type.
+
+#### Storage
+
+The pallet stores identity commitments, which are opaque byte blobs in the pallet storage and on which you can build the cross-chain identity bridging protocol.
```

Comment:
Did we specify who "you" is in this case? We are targeting runtime developers, but I did not read that anywhere.